test/fixtures/doc-server/features/steps/steps.rb

```diff
@@ -9,7 +9,7 @@
   assert_compare millis, '<', duration
 end
 
-When('I make a reflective {word} request on port {string} with status {string} and delay of {string}') do |method, port, status, delay_ms|
+When('I make a reflective {string} request on port {string} with status {string} and delay of {string}') do |method, port, status, delay_ms|
```

Comment:
I needed to change to a `{string}` here as apparently Cucumber doesn't allow hyphens in a `{word}`.